### ISSUE TITLE
Add support for SASS

### ISSUE THREAD

**Issue by stpaultim** [backdrop-contrib/tatsu#6]
Let's add SASS support to this theme.

**Comment by yorkshire-pudding** [comment]
Having looked into SASS, I'm intrigued by what this means.  Does this mean providing the sass/scss files so subthemers can use these as the basis for overriding, just to make it easier to manage or would sass be run from settings?  Apologies if these are daft questions, but I don't know quite what is possible with sass yet.

**Comment by stpaultim** [comment]
@yorkshire-pudding - Nothing daft about your questions. Anyone can use SASS on their own with any theme if they wish to take the time to set up the tooling and configuration. 

I got started using SASS by using the Zen theme that came with detailed instructions, starter files, and configuration for the tooling included in the sub-theme starter kit. The tooling and instructions assumed that the user would be using Gulp to manage and compress their SASS.

In my view, Tatsu could provide a SASS starter kit by doing the same. Providing all the files and instructions on how to use SASS with a sub-theme to make it much easier for beginners. 

So, yes. Support for SASS could include providing files that can be overwritten, but also by providing some tooling configuration and instructions to make it easy.

Here are the instructions I often refer to in regards to using SASS and Gulp in a Drupal or Backdrop theme:
https://chenhuijing.com/blog/drupal-101-theming-with-gulp/

**Comment by yorkshire-pudding** [comment]
Thanks.  I'm setup with just the standalone tool for now.  I'm wondering whether there is also scope to improve the performance of the theme by converting the component files to SCSS and compiling to a single file?  Even with CSS aggregation selected, there still seems to be a large number of css files.

**Comment by izmeez** [comment]
We used SASS a lot with zen themes in drupal and enjoyed it. Trying to identify the benefits of SASS:

1. Compresses all css files into a single minimized styles.css file.
2. Use of zen-grids to fine tune breakpoints and flow. The nice thing with zen is the media queries were all in one file called layouts.css which is an obvious departure for backdrop. But so far the few backdrop themes we've looked at have media queries scattered throughout. This makes it much harder to see them all together and to make changes.
3. Using Compass and SASS for font sizes and color attributes. This was interesting could it could be defined in one place at the start and making a simple change there would be applied throughout. The use of a color model for branding was interesting and useful. http://blog.12spokes.com/web-design-development/simple-css-color-management-with-sass/ 
4. Unfortunately, the more use made of these features, the greater the burden when transitioning to different platforms especially if Compass & SASS are not available.

**Comment by Wylbur** [comment]
We'll need some action on this before we can add this to the current release, so removing this from the impending 1.2 release.

**Comment by stpaultim** [comment]
Yes. I think that SASS is becoming a bit less popular as some of it's nicest features are becoming available in pure CSS. Things like variables. 

This issue has been tabled for now, but remains open for comment.

**Comment by izmeez** [comment]
I'm also not sure that SASS is important.
